Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkMiniPlayer.cs

```diff
@@ -50,15 +50,26 @@ public class AudioLinkMiniPlayer : UdonSharpBehaviour
         int _syncVideoNumber;
         int _loadedVideoNumber;
 
-        [UdonSynced, NonSerialized]
+        /// <summary>
+        /// Synced via <see cref="_flags"/>
+        /// </summary>
+        [NonSerialized]
         public bool _syncOwnerPlaying;
 
         [UdonSynced]
         float _syncVideoStartNetworkTime;
 
-        [UdonSynced]
+        /// <summary>
+        /// Synced via <see cref="_flags"/>
+        /// </summary>
         bool _syncLocked = true;
 
+        /// <summary>
+        /// Flags byte for various settings that are synced. Order is <see cref="_syncOwnerPlaying"/>, <see cref="_syncLocked"/>
+        /// </summary>
+        [UdonSynced]
+        byte _flags = 0;
```

Comment:
Maybe make the default value for this match the default value for the fields? ie 0b10

Reply:
I changed it to that, however its not a big deal as it is also set in start() to ensure that it is correct